napari/_qt/dialogs/qt_plugin_dialog.py

```diff
@@ -62,7 +62,7 @@ def __init__(
     ):
         super().__init__()
         self._queue = []
-        self._processes = []
+        self._processes = {}
```

Comment:
```suggestion
        self._queue: List[Tuple[Tuple[str, ...], Callable[[], QProcess]]] = []
        self._processes: Dict[Tuple[str, ...], QProcess] = {}
```

optional... but might be nice to quickly see what these are supposed to hold as they get a bit more complicated.  You'll need some more imports at the top too

also... maybe use a `collections.deque` now since we're mostly just inserting and popping from the ends?

Reply:
Fixed!